.ci/check-packages.groovy

```diff
@@ -99,7 +108,7 @@ pipeline {
   }
   post {
     cleanup {
-      notifyBuildResult(prComment: true)
+      notifyBuildResult(prComment: false)
```

Comment:
This particular notification won't happen anyway, since this will only work for branches no PRs